src/apps_plugin.c

```diff
@@ -824,13 +867,15 @@ int read_proc_pid_stat(struct pid_stat *p) {
 
 	// ----------------------------------------
 
-	int set_quotes = (!ff)?1:0;
+	// FIX: set_quotes will always be true!
+	//int set_quotes = (!ff)?1:0;
```

Comment:
no it will not. `ff` is static. The whole idea with these `static profile *ff` is that they maintain their internal structures among calls. They adapt their internal structures to the sizes of the files they see.